src/v/kafka/server/handlers/sasl_authenticate.cc

```diff
@@ -20,7 +21,7 @@ ss::future<response_ptr> sasl_authenticate_handler::handle(
   request_context ctx, [[maybe_unused]] ss::smp_service_group g) {
     sasl_authenticate_request request;
     request.decode(ctx.reader(), ctx.header().version);
-    vlog(klog.debug, "Received SASL_AUTHENTICATE {}", request);
+    vlog(klog.debug, "Received SASL_AUTHENTICATE");
```

Comment:
should we also remove (or alter to obfuscate data) the `operator<<` overload for `sasl_authenticate_request` so any future output of the request is not logged with the bytes payload? the same with `sasl_authenticate_response`, which also contains auth bytes payload and is currently automatically logged here: https://github.com/redpanda-data/redpanda/blob/dev/src/v/kafka/server/request_context.h#L154

Reply:
Yeah, I removed the response bytes from the log statement you linked to, and to be sure, went the route you suggested of redacting some message types.

Reply:
Done